src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1482,7 +1482,7 @@ private void VisitNullChecks(IOperation operation, ImmutableArray<IParameterSymb
         private ConditionalOperation GenerateNullCheckForParameter(IParameterSymbol parameter, SyntaxNode syntax, SemanticModel semanticModel)
         {
             Debug.Assert(parameter.Language == LanguageNames.CSharp);
-            Optional<object> constantValue = default(Optional<object>);
+            ConstantValue constantValue = null;
```

Comment:
Actually, this is the source of the IOperation issue. It should be `ConstantValue.Null`: http://sourceroslyn.io/#Microsoft.CodeAnalysis/ConstantValue.cs,d8af96f1a97db377,references.

Reply:
Thanks, updated as such.